src/Homework7/Task5.java

```diff
@@ -0,0 +1,37 @@
+package Homework7;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Task5 {
+        public static void main(String[] args) {
+            try {
+                String text = "";
```

Comment:
It can be replaced with `StringBuilder` to `append` strings in the loop on line 16.

Reply:
Done.